### PR TITLE
chore: updated near-* to 0.24.0

### PR DESCRIPTION
**Pull request by akorchyn**
CC @race-of-sloths

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -19,10 +19,10 @@ thiserror = "1.0.37"
...
 lazy_static = "1.4.0"
 
-near-crypto = ">0.22,<0.24"
-near-primitives = ">0.22,<0.24"
-near-chain-configs = ">0.22,<0.24"
-near-jsonrpc-primitives = ">0.22,<0.24"
+near-crypto = ">0.22,<0.25"
+near-primitives = ">0.22,<0.25"
+near-chain-configs = ">0.22,<0.25"
+near-jsonrpc-primitives = ">0.22,<0.25"
```
Comment (frol) [inline Cargo.toml:25]: I feel that the changes in the code (e.g. `TransactionV0`) mean that older versions of the crates won't work, will they?

Comment (akorchyn) [reply Cargo.toml:25]: It compiles with older version (you can see that I haven't changed the lib itself)
We don't use Cargo.lock, that's why we would end up with latest 0.24 version, and it has some changes in API :)

Though, I'm not so sure about semver.

File: examples/create_account.rs [hunk 2]
```diff
@@ -228,7 +230,7 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
                     }
                     FinalExecutionStatus::SuccessValue(ref s) => {
                         // outcome.status != SuccessValue(`false`)
-                        if s == b"false" {
+                        if s.is_empty() {
```
Comment (frol) [inline examples/create_account.rs:233]: I am surprised by both previous `false` check and `is_empty` check now.

`on_account_create` finishes with a boolean output `true` if account was successfully created: https://github.com/near/near-linkdrop/blob/49279e529c254fa7736465b4a39d05cb8f1e5443/src/lib.rs#L211

File: Cargo.toml (hunk 1 above)
Comment (frol) [reply Cargo.toml:25]: Ah, those are examples. Sorry, it is fine indeed

File: examples/create_account.rs (hunk 2 above)
Comment (akorchyn) [reply examples/create_account.rs:233]: Kinda weird, but I have run it manually and it returned me ""

Comment (akorchyn) [reply examples/create_account.rs:233]: Maybe it because I have tried to create from myself (sub account)

Comment (akorchyn) [reply examples/create_account.rs:233]: Confirmed. It return nothing if you are creating subaccount, and returns true if you are creating from linkdrop

**Comment by race-of-sloths** [comment]
@akorchyn Thank you for your contribution! Your pull request is now a part of the [Race of Sloths](https://race-of-sloths.com)!
Are you going to win race-of-sloths this month? If so, you should speed up! Run, Sloth, run!

[<picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://badge.race-of-sloths.com/akorchyn?type=bot&theme=dark">
    <source media="(prefers-color-scheme: light)" srcset="https://badge.race-of-sloths.com/akorchyn?type=bot&theme=light">
    Shows profile picture for the author of the PR
</picture>](https://race-of-sloths.com/profile/akorchyn)


<details><summary>Current status: <i>waiting for scoring</i></summary>

We're waiting for maintainer to score this pull request with `@race-of-sloths score [0,1,2,3,5,8,13]` command. Alternatively, autoscoring [1,2] will be applied for this pull request

</details><details><summary>What is the Race of Sloths</summary>
<p>
Race of Sloths is a friendly competition where you can participate in challenges and compete with other open-source contributors within your normal workflow

For contributors:

- Tag @race-of-sloths inside your pull requests
- Wait for the maintainer to review and score your pull request
- Check out your position in the [Leaderboard](https://race-of-sloths.com/leaderboard)
- Keep weekly and monthly streaks to reach higher positions
- Boast your contributions with a dynamic picture of your [Profile](https://race-of-sloths.com/profile/akorchyn)

For maintainers:
- Score pull requests that participate in the Race of Sloths
- Engage contributors with fair scoring and fast responses so they keep their streaks
- Promote the Race to the point where the Race starts promoting you
- Grow the community of your contributors

Feel free to check [our website](https://race-of-sloths.com) for additional details!
</p>
</details> 

<details>
<summary>Bot commands</summary>

- For contributors
  - **Include a PR:** `@race-of-sloths include` to enter the Race with your PR
- For maintainers:
  - **Invite contributor** `@race-of-sloths invite` to invite the contributor to participate in a race or include it, if it's already a runner.
  - **Assign points:** `@race-of-sloths score [1/2/3/5/8/13]` to award points based on your assessment.
  - **Reject this PR:** `@race-of-sloths exclude` to send this PR back to the drawing board.
  - **Exclude repo:** `@race-of-sloths pause` to stop bot activity in this repo until `@race-of-sloths unpause` command is called

</details>

**Comment by frol** [comment]
@race-of-sloths score 3

**Pull request merged**